docs/sdk/src/reference_docs/omni_node.rs

```diff
@@ -0,0 +1,170 @@
+//! # (Omni) Node
+//!
+//! This reference docs elaborates on what a Polkadot-SDK/Substrate node software is, and what
+//! various ways exist to run one.
+//!
+//! The node software, as denoted in [`crate::reference_docs::wasm_meta_protocol`], is everything in
+//! a blockchain other than the WASM runtime. It contains common components such as the database,
+//! networking, RPC server and consensus. Substrate-based nodes are native binaries that are
+//! compiled down from the Rust source code. The `node` folder in any of the [`templates`] are
+//! examples of this source.
+//!
+//! <details>
+//! <summary>
+//! A typical node also contains a lot of other tools (exposed as subcommands) that are useful for
+//! operating a blockchain:
+//! </summary>
+// #![doc = docify::embed!("../../substrate/bin/node/cli/src/cli.rs", "Subcommand")]
```

Comment:
This does not render correctly (no details are expanded).

Reply:
Yeah, I could not get docify to work in solochain tempalte for some reason, I will try again now.